### ISSUE TITLE
Refresh Contributor Day page in preparation for November 2023 Hack Day

### ISSUE THREAD

**Issue by danielbachhuber** [wp-cli/wp-cli#5856]
**Done is:**
* The page is refreshed.
* We've tested the page contents with a user or two.

**Comment by wojtekn** [comment]
@danielbachhuber I went through the setup instructions, my notes are below:

- I needed to create a repository token with read access for the repo to make `composer install` work
- As I use Docker image for database, I needed to export the host and password to make `composer prepare-tests` work:

```
export WP_CLI_TEST_DBHOST=127.0.0.1
export WP_CLI_TEST_DBROOTPASS=password1
composer prepare-tests
```

I created a Docker container with the following command:

```
docker run --name wpcli-hackday-mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=password1 -d circleci/mariadb:10.3.23
```

It took me some time to figure it out. Maybe we could link to https://github.com/wp-cli/wp-cli-tests#the-database-credentials or repeat those instructions in the Hack Day doc?

With all those done, I can run the `composer test` in the repository root, and the `lint` and `phpcs` tests were executed. I'm unsure how to run PHPUnit tests in this flow, though.

Do you think we should provide an example showing how to run tests? And maybe another one showing how to run the CLI command the developer is working on?

**Comment by sejas** [comment]
**Setting up the environment**
I cloned the repo, copied and pasted the commands, and it worked everything fine on my end. Great instructions !!
I also tried it running some tests. See https://github.com/wp-cli/wp-cli/issues/5858#issuecomment-1803602961

```
git clone git@github.com:wp-cli/wp-cli-dev.git
cd wp-cli-dev
composer install
composer prepare-tests
```

**Add information about the open video sessions**

I think some information from https://make.wordpress.org/cli/2023/10/18/save-the-date-wp-cli-hack-day-on-friday-november-10th/ would be important to be highlighted in the readme. Specifically the times for the open video sessions.

```
- From 🕓 08:00 WET to 09:00 WET
- From 🕓 15:00 WET to 16:00 WET
```

- https://github.com/wp-cli/handbook/pull/473

**Add a `Thanks for contributing` section**

I feel the Readme needs a conclusion/closing section.
- Here is my suggestion https://github.com/wp-cli/handbook/pull/472

**Comment by kozer** [comment]
I also followed the instructions and it were very clear till the:
```
composer prepare-tests
```
command.
Those are the issues I came across:
1. I didn't have `mysql` installed. So I got an error about that at first.
2. For some reason I couldn't connect as a `user` until I ran it with `sudo`. That made me to create a new user like so:
```
CREATE USER 'monty'@'localhost' IDENTIFIED BY 'some_pass';
GRANT ALL PRIVILEGES ON *.* TO 'monty'@'localhost' WITH GRANT OPTION;
```
Then I exported the new user and the password:
```
export WP_CLI_TEST_DBROOTPASS=some_pass
export WP_CLI_TEST_DBROOTUSER=monty
```
**Notes about my environment**
1. I'm using [Arch Linux](https://wiki.archlinux.org/title/MariaDB)
2. Php is installed using phpenv. I installed php env using [phpenv-installer](https://github.com/phpenv/phpenv-installer)

**Suggestions**
It could be a good ticket to add a  `docker-compose` script, or a `Dockerfile` to spin up a MySQL db, and possibly a bash script to install `phpenv` in order to offload the user of having to install anything.
